gstd/gstd_chmod.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+# This script is executed at install time to change the mode of the directories installed
+
+chmod 777 "${DESTDIR}/${MESON_INSTALL_PREFIX}/$1"
```

Comment:
I recommend receiving the permission as an argument as well.

Reply:
Done